<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkCurvedMPRViewControls</class>
 <widget class="QWidget" name="QmitkCurvedMPRViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>1028</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QComboBox" name="PlaneComboBox">
     <item>
      <property name="text">
       <string>xy_plane</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>xz_plane</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>yz_plane</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sphere</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prefit plane</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Stiffness (&lt;font face=&quot;Symbol&quot;&gt;s&lt;/font&gt; [%])</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="SigmaSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SigmaSpinBox">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="displayBox">
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QGroupBox" name="DisplayMode">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rb3D">
           <property name="text">
            <string>3D</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb2D">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>2D</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="ReformationType" native="true">
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rbCurved">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>curved reformation</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbCrossSection">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>cross section</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QmitkSliderNavigatorWidget" name="crossSectionNavigator" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showPoints">
        <property name="text">
         <string>show points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showCurvedReformat">
        <property name="text">
         <string>show reformat in 3D</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pointsBox" native="true">
     <layout class="QVBoxLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QmitkPointListWidget" name="pointListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ioBox">
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="GetPointsFromDataTreeBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Get Points From Data Tree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LoadPoints">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load Points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SavePoints">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save Points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveWidget4">
        <property name="text">
         <string>Save Result View</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cutBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cbAutoOutsideValue">
        <property name="text">
         <string>automatic outside value</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="lbOutsideValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>outside value</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbOutsideValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QListView</extends>
   <header>QmitkPointListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkSliderNavigatorWidget</class>
   <extends>QWidget</extends>
   <header>QmitkSliderNavigatorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">QmitkPointListWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>SigmaSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SigmaSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SigmaSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SigmaSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SigmaSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QmitkCurvedMPRViewControls</receiver>
   <slot>SigmaSpinBox_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
